### PR TITLE
Dedicated access keys for hmpps-domain-events-dev topic

### PR DESCRIPTION
**Pull request by marcus-bcl**

### CODE REVIEW

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/variables.tf [hunk 1]
```diff
@@ -19,6 +19,34 @@ variable "team_name" {
...
+    "hmpps-tier-dev",
+    "hmpps-workload-dev",
+    "make-recall-decision-dev",
+    "offender-case-notes-dev",
+    "offender-events-dev",
+    "offender-management-staging",
+    "offender-management-test",
+    "offender-management-test2",
+    "prisoner-offender-search-dev"
+  ]
```
Comment (marcus-bcl) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/variables.tf:47]: Anyone who needs access to the hmpps-domain-events topic just needs to add their namespace to this list, which will then create a dedicated access key and store it in a secret named "hmpps-domain-events-topic".

Comment (marcus-bcl) [reply namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/variables.tf:47]: There are a couple actually, e.g. https://github.com/ministryofjustice/cloud-platform-environments/blob/dedicated-access-keys-dev/namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/hmpps-activities-management.tf#L119-L130.

For those ones I've left the old secret in place, and created them a new one with the standard name.

Comment (marcus-bcl) [reply namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/variables.tf:47]: The ones with non-standard secret names are:
* activities-api-dev - "activities-domain-events-sqs-topic-instance-output"
* offender-case-notes-dev - "domain-events-topic"
* hmpps-workload-dev - "wmt-hmpps-domain-events-topic"

Comment (timharrison-moj) [reply namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/variables.tf:47]: I've created a ticket to rename the activities-api-dev secret to be consistent, and will remove the old one too. Thanks.

**Review by jakemulley — APPROVED** [review]

**Pull request merged**